### PR TITLE
First look to Asset feature.

### PR DESCRIPTION
**Pull request by JRojas999**

### CODE REVIEW

File: cypress/src/core/action.js [hunk 1]
```diff
@@ -4,6 +4,12 @@ export function setField(selector, inputText) {
 export function clickField(selector) {
     cy.get(selector).click()
 }
+export function checkField(selector) {
+    cy.get(selector).check()
```
Comment (jpgonzales27) [inline cypress/src/core/action.js:10]: Maybe these function should be a space blank each other

**Review by jpgonzales27 — APPROVED** [review]

File: .eslintrc.json [hunk 2]
```diff
@@ -178,7 +178,7 @@
         "object-curly-spacing": "error",
         "object-property-newline": "error",
         "object-shorthand": "error",
-        "one-var": "error",
+        "one-var": "off",
```
Comment (edsonarios) [inline .eslintrc.json:181]: What was the reason for disabling this rule?

Comment (JRojas999) [reply .eslintrc.json:181]: Because only allows one variable per method.

File: cypress/integration/asset/create-asset.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        createAsset(asset)
+
+        validateAsset(asset)
+    })
+
+    it('should create it with all attributes', () => {
+        const asset = {
+            "account": 'cuenta 13',
+            "competitor": true,
+            /* "contact": 'contact2', */
```
Comment (edsonarios) [inline cypress/integration/asset/create-asset.js]: This comment should be erase

File: cypress/src/core/action.js (hunk 1 above)
Comment (JRojas999) [reply cypress/src/core/action.js:10]: I agree with you.

File: cypress/integration/asset/create-asset.js (hunk 3 above)
Comment (JRojas999) [reply cypress/integration/asset/create-asset.js]: It's right.

**Review by edsonarios — APPROVED** [review]

**Pull request merged**